backend/main.go

```diff
@@ -417,7 +417,9 @@ func main() {
 			zapier.CreateZapierRoutes(r, db, &zapierStore, &rh)
 		})
 		r.HandleFunc("/slack-events", privateResolver.SlackEventsWebhook(ctx, slackSigningSecret))
+		r.Post(fmt.Sprintf("%s/%s", privateEndpoint, "microsoft-teams/bot"), privateResolver.MicrosoftTeamsBotEndpoint)
```

Comment:
An endpoint it needed by the microsoft bot framework when setting up a bot.
This is used by the framework to send messages to the bot.

All request validation is done by the msbotbuilder framework